.github/CODEOWNERS

```diff
@@ -7,4 +7,4 @@
 # someone opens a pull request.
 
 # Everything requires team review by default
-* @badhrinathpa @pudelkoM @cgoea @ajaythakurintel @adibrastegarnia @SeanCondon @scottmbaker @guptagunjan @sys-devops-approve
+* @scottmbaker @soniabha-intc @hyunsun @cjnolan @rranjan3 @SushilLakra @sys-devops-approve
```

Comment:
Should not remove @guptagunjan as codeowner
Should add @krishnajs and @nathsudi